Microsoft.Research/Clousot.Cache/SQLCacheModel.cs

```diff
@@ -784,5 +786,33 @@ protected override SqlConnectionStringBuilder BuildConnectionString(IClousotCach
 
       return base.BuildConnectionString(options);
     }
+
+    public static string GetDesiredDbName()
+    {
+        var versionKeyFileMapping = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
```

Comment:
That's why I did it like this - now the other client is no longer broken.
Currently the responsibility of the ```LocalDbClousotCacheFactory``` is to provide a connection to LocalDB - no matter what engine is installed.
If the client ever will need to take responsibility of the database, it can using the proper constructor.